tests/view_tests/tests/test_debug.py

```diff
@@ -380,6 +381,30 @@ def test_unfrozen_importlib(self):
         html = reporter.get_traceback_html()
         self.assertIn('<h1>ImportError at /test_view/</h1>', html)
 
+    def test_ignore_traceback_evaluation_exceptions(self):
+        """
+        Don't trip over exceptions generated by crafted objects when
+        evaluating them while cleansing. Refs #24455.
+        """
+        request = self.rf.get('/test_view/')
+        broken_lazy = SimpleLazyObject(lambda: 0 / 0)
```

Comment:
I think we should trigger a custom exception here rather than `ZeroDivisionError`